src/main/java/frc/robot/commands/UpAndDown.java

```diff
@@ -0,0 +1,35 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.Climb;
+
+public class UpAndDown extends Command {
+    private Climb climb;
+    private boolean direction;
+
+    public UpAndDown(Climb climb, boolean direction) {
+        this.climb = climb;
+        this.direction= direction;
+        addRequirements(climb);
+    }
+
+    @Override
+    public void initialize() {
+
+    }
+
+    @Override
+    public void execute() {
+        climb.rotateUpForward(direction);
+    }
+
+    @Override
+    public void end(boolean interrupted) {
+        climb.stop();
+    }
+
+    @Override
+    public boolean isFinished() {
```

Comment:
use the limit switches to know when to stop the command.

Reply:
When moving up we want the forward limit switch to stop the movement and when moving down the reveres one need to stop it. 
For example, if I wish to raise the system, only when reaching the upper limit switch, the command should stop.